libs/sx1302_hal/Makefile

```diff
@@ -0,0 +1,116 @@
+#
+# Copyright (C) 2022 TDT AG <development@tdt.de>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+#
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=sx1302_hal
+PKG_VERSION:=2.1.0
+PKG_RELEASE:=$(AUTORELEASE)
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/Lora-net/sx1302_hal/tar.gz/V$(PKG_VERSION)?
+PKG_HASH:=4b62aa6a83ad449c68fdd844fc35024586c9faabca3c5a90a2544735b380de5b
+
+PKG_MAINTAINER:= Marcus Schref <mschref@tdt.de>
+PKG_LICENSE:=MIT BSD-3-Clause
+PKG_LICENSE_FILES:=LICENSE.TXT
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/sx1302_hal
+  SECTION:=libs
+  CATEGORY:=Libraries
+  TITLE:=SX1302/SX1303 HAL
+endef
+
+define Package/sx1302_hal/description
+  SX1302/SX1303 Hardware Abstraction Layer
+endef
+
+define Package/sx1302_hal-tests
+  SECTION:=net
+  CATEGORY:=Network
+  SUBMENU:=LoRaWAN
+  TITLE:=Tests for SX1302/SX1303 HAL
```

Comment:
Shouldn't be those test in one category? Right now, it is in two categories in ``make menuconfig``. What about something like this or vice versa?

```
define Package/sx1302_hal-tests
  $(call Package/sx1302_hal)
  TITLE:+=(Tests)
  DEPENDS:=sx1302_hal +kmod-usb-acm
endef
```

Also, there is a missing submenu for the first package.

Reply:
You are right. All three packages are now located at -> Libraries -> IoT.